packages/dom/test/helpers.js

```diff
@@ -1,5 +1,5 @@
 // create and cleanup testing DOM
-export function withExample(html) {
+export function withExample(html, shadow = true) {
```

Comment:
let's pass `options` instead of `shadow` we can deflate these options to bring more options in the future.